condor_templates_test.go

```diff
@@ -108,8 +108,6 @@ func TestGenerateCondorSubmit(t *testing.T) {
 executable = /usr/local/bin/road-runner
 rank = 100 - TotalLoadAvg
 requirements = HasDocker && (HAS_CYVERSE_ROAD_RUNNER =?= True) && (HAS_HOST_MOUNTS =?= True)
-request_cpus = 4.5
-request_memory = 2048MB
```

Comment:
It might still be a good idea to test that these values are getting populated correctly in the template.

So rather than remove these from the expected output, maybe we should update the test JSON with the correct fields for these values?
`max_cpu_cores` ➡️ `min_cpu_cores`
`memory_limit` ➡️ `min_memory_limit`

I think they need to be updated only in `extra_requirements_submission.json`, `no_volumes_submission.json`, and `test_submission.json`.

Reply:
That's a good idea. maybe what I'll do is add a separate test with the `min_cpu_cores` and `min_memory_limit` settings instead. That way, we can test both cases.

Reply:
Good idea.

Reply:
I just pushed the commit with the additional unit test.